src/hash_table.js

```diff
@@ -0,0 +1,173 @@
+class HashEntry {
+  constructor(key, value) {
+    this.key = key
+    this.value = value
+    this.next = null
+  }
+}
+export default class HashTable {
+
+  constructor() {
+    this.currentSize = 0
+    this.storageMax = 128
+    this.storage = []
+  }
+
+  put(key, value) {
+    let hash = HashTable.hash(key) % this.storageMax
```

Comment:
you can do `this.hash()` here, cause it seems odd to me to call `HashTable.hash()`